Applications/VpView/vpFseTrackIO.cxx

```diff
@@ -255,8 +269,21 @@ bool vpFseTrackIO::ImportTracks(vtkIdType idsOffset,
 }
 
 //-----------------------------------------------------------------------------
-bool vpFseTrackIO::WriteTracks(const char* filename,
-                               bool writeSceneElements) const
+QStringList vpFseTrackIO::GetSupportedFormats() const
+{
+  return {"Scene Elements (JSON) (*.json)"};
```

Comment:
Building with VisualStudio 2017 (note, this branch requires Qt4):
error C3445: copy-list-initialization of 'QStringList' cannot use an explicit constructor

so, 
return QStringList("Scene Elements (JSON) (*.json)");

?

Reply:
Turns out I was building from a slightly old branch (0a687faf6). However, now I'm having other build issues:

4>c:\vision\diva\vivia\build\include\vvvtkwidgets\ui_videoQuery.h(1): fatal error C1083: Cannot open include file: 'C:/Vision/DIVA/vivia/build/Libraries/VvVtkWidgets/ui_videoQuery.h': No such file or directory
c:\vision\diva\vivia\build\include\vvvtkwidgets\ui_videoView.h(1): fatal error C1083: Cannot open include file: 'C:/Vision/DIVA/vivia/build/Libraries/VvVtkWidgets/ui_videoView.h': No such file or directory


The missing include files are deeper down the expected path:
C:\Vision\DIVA\vivia\build\Libraries\VvVtkWidgets\vvVtkWidgets_autogen\include_Release

Reply:
This was fixed by #64. I would either build with the branch merged into latest master, or else copy the generated headers to the "expected" location. (I had old instances of the headers lying around, which is why the problem went unnoticed when #62 went in.)